eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -15,7 +15,8 @@
     <UsagePattern IdentityGlob="System.Security.Principal.Windows/4.5.0" />
     <UsagePattern IdentityGlob="Microsoft.Diagnostics.NETCore.Client/0.2.452401" />
     <!-- Caused by multi-targeting source build for repo builds/CI. Product source build only targets NetCurrent. -->
-    <UsagePattern IdentityGlob="Microsoft.AspNetCore.App.Ref/8.0.0" />
-    <UsagePattern IdentityGlob="Microsoft.NETCore.App.Host.linux-x64/8.0.0" />
+    <UsagePattern IdentityGlob="Microsoft.AspNetCore.App.Ref/8.0.1" />
```

Comment:
If you need to unblock this PR, please disable the dependency flow into SDK until https://github.com/microsoft/vstest/issues/4927 is fixed otherwise if this flows, it will break source-build and will block the sdk->installer dependency flow until resolved.

Reply:
(y)

Reply:
@MichaelSimons  I don't see any way for us to eliminate this. Would it work (in theory) if we run 2 source build runs, 1 for each target framework? We are using an arcade template that does not allow this right now, but if we made it work, would it solve our issue for source-build? I don't know what is happening on the background.

Reply:
I took a second look at this.  Because vstest flows into multiple SDK versions it is impossible with our current infra to get the repo level source-build to not have the 8.0 related prebuilts.  This shouldn't be an issue in the product source-build because of [this condition](https://github.com/microsoft/vstest/blob/main/Directory.Build.targets#L18).  

To make your code more sustainable, this UsagePattern entry should generalized so that it doesn't have to be updated constantly.  My suggestion is to change 8.0.1 to 8.0.*